sdk/metricsadvisor/ai-metrics-advisor/src/models.ts

```diff
@@ -774,14 +774,14 @@ export type DataFeedPatch = {
   accessMode?: DataFeedAccessMode;
 
   /**
-   * email addresses of data feed administrators
+   * data feed administrators email addresses or client id
    */
-  adminEmails?: string[];
+  admins?: string[];
```

Comment:
What does a client id look like/where does it come from? I'm surprised it's all strings, is the service doing some pattern matching on the strings to determine whether it is an email address or something else?

Reply:
Client id is more like a GUID. consider the example of using AAD authentication - where we use client secret credential - we can use the client id used there to give that user access as a viewer or add as an admin. I am not sure how service processes it, I can ask the service team.

Reply:
I'm wondering if we can document where to get the client id from.

Reply:
It should be the same that we use for AAD - 
https://docs.microsoft.com/en-us/azure/active-directory/azuread-dev/v1-oauth2-client-creds-grant-flow#first-case-access-token-request-with-a-shared-secret

Reply:
`data feed viewers email addresses or application client id` - Is this more clear?

Reply:
@chradek I pinged you the thread on Teams where we are discussing with the service team about what values these fields accept

Reply:
I updated the ref-docs for admins and viewers